client/src/components/RatingsReviews.jsx

```diff
@@ -45,7 +45,7 @@ export default function RatingsReviews({ productID, reviewMetadata }) {
   };
 
   return (
-    <div className="ratings-reviews">
+    <div className="ratings-reviews" ref={reviewRef}>
```

Comment:
dope. so you're just storing a reference to this specific DOM node so you can scroll to it from your widget. cool